.travis.yml

```diff
@@ -18,8 +18,10 @@ before_install:
 - echo HRNGDEVICE=/dev/urandom | sudo tee /etc/default/rng-tools
 - sudo /etc/init.d/rng-tools restart
 install:
-- pip install ansible==1.8.4
-- pip install coveralls
+- pip freeze -l
+# Preinstalled travis pytest is not tracked by us
+- pip uninstall pytest -y
+- pip install -r testinfra/requirements.txt
```

Comment:
I think it would be better to merge this file with `securedrop/requirements/test-requirements.in` and just let the existing role that installs the test deps install these. Even if we don't always need all of them in the course of development, it shouldn't add a very significant amount of time to provisioning. This way we also don't have to worry about version clash. You could also remove the `install` section here entirely as long as you are installing with `pip install -U -r <requirements file>`. Since `pip-tools` fixes the lower-level requirements in the `.txt` file it produces, one doesn't have to worry about `-U` producing different results if non-top-level requirements are updated on PyPi.